index.html

```diff
@@ -1371,6 +1389,67 @@ <h3 id="name-requesting-a-credential-usi">
+        </h3>
+<p id="section-2.6-1"><a href="https://w3c.github.io/did-core/">Decentralized identifiers</a> are a resolvable identifier to a set of statements about the [did subject]() including a set of cryptographic material (e.g public keys). Using this cryptographic material, a <a href="https://w3c.github.io/did-core/">decentralized identifier</a> can be used as an authenticatable identifier in a credential.<a href="#section-2.6-1" class="pilcrow">¶</a></p>
+<p id="section-2.6-2">The below section highlights how a client construct a credential request to obtain a credential that is bound to the client via the usage of a <a href="https://w3c.github.io/did-core/">decentralized identifier</a>.<a href="#section-2.6-2" class="pilcrow">¶</a></p>
+</section>
+</div>
+<div id="credential-request-using-a-decentralized-identifier">
+<section id="section-2.7">
+        <h3 id="name-credential-request-using-a-">
+<a href="#section-2.7" class="section-number selfRef">2.7. </a><a href="#name-credential-request-using-a-" class="section-name selfRef">Credential Request using a Decentralized Identifier</a>
+        </h3>
+<p id="section-2.7-1">A Client can request in the credential issuance flow, that the resulting credential be bound to the client through the usage of <a href="https://w3c.github.io/did-core/">decentralized identifiers</a> by using the <code>did</code> field.<a href="#section-2.7-1" class="pilcrow">¶</a></p>
+<p id="section-2.7-2">An OpenID Provider processing a credential request featuring a <a href="https://w3c.github.io/did-core/">decentralized identifier</a> MUST follow the following additional steps to validate the request.<a href="#section-2.7-2" class="pilcrow">¶</a></p>
+<ol start="1" type="1" class="normal type-1" id="section-2.7-3">
+          <li id="section-2.7-3.1">
+            <p id="section-2.7-3.1.1">Validate the value in the <code>did</code> field is a valid <a href="https://w3c.github.io/did-core/">decentralized identifier</a><a href="#section-2.7-3.1.1" class="pilcrow">¶</a></p>
+</li>
+<li id="section-2.7-3.2">
+            <p id="section-2.7-3.2.1">Resolve this the <code>did</code> value to a [did document]().<a href="#section-2.7-3.2.1" class="pilcrow">¶</a></p>
+</li>
+<li id="section-2.7-3.3">
+            <p id="section-2.7-3.3.1">Validate that the key in the <code>sub_jwk</code> field of the request appears in the <code>publicKey</code> section of the [DID Document]().<a href="#section-2.7-3.3.1" class="pilcrow">¶</a></p>
+</li>
+</ol>
+<p id="section-2.7-4">If any of the steps fail then the OpenID Provider MUST respond to the request with the Error Response parameter, <a href="https://openid.net/specs/openid-connect-core-1_0.html#AuthError">section 3.1.2.6.</a> with Error code: <code>invalid_did</code>.<a href="#section-2.7-4" class="pilcrow">¶</a></p>
```

Comment:
This makes sense, but I'm not sure how the behavior defined above around ignoring did if it's not included could interact with this to cause unknown behavior. I can't think of a particular problem at the moment with it, but want to call it out so we're aware of it and getting more certainty around the topic.